.github/workflows/deploy-pages.yml

```
@@ -54,36 +64,70 @@ jobs:
         with:
           name: conformance-${{ steps.get-details.outputs.name }}.json
           path: ./output.json
-  aggregate:
-    runs-on: "ubuntu-latest"
+  # https://github.com/actions/starter-workflows/blob/4a8f18e34dd13d2b6ee4d8da2ba72629eafe1609/pages/hugo.yml#L1
+  build:
+    runs-on: ubuntu-latest
+    env:
+      HUGO_VERSION: 0.117.0
     needs: [pull-outputs]
-    # the tests might have failed
-    if: always()
-    defaults:
-      run:
-        shell: bash
     steps:
-      - uses: actions/checkout@v3
+      - name: Setup Hugo
+        uses: peaceiris/actions-hugo@v2
```

Let's pin this to sha.

```suggestion
        uses: peaceiris/actions-hugo@16361eb4acea8698b220b76c0d4e84e1fd22c61d # v2.6.0
```